plugins/automox/plugin.spec.yaml

```diff
@@ -1,19 +1,19 @@
 plugin_spec_version: v2
 extension: plugin
-products: [insightconnect]
+products: [ insightconnect ]
 name: automox
 title: Automox
 description: Automox is modernizing IT operations with continuous visibility, insight, and agility for your entire IT environment
-version: 1.2.0
-supported_versions: ["All as of 1/21/2022"]
+version: 2.0.0
+supported_versions: [ "All as of 10/09/2023" ]
 vendor: automox
 support: partner
```

Comment:
Can SDK please be added here and the version updated to the latest.

Example:
sdk:
  type: slim
  version: 5
  user: nobody